packages/eslint-config-airbnb-base/rules/imports.js

```diff
@@ -68,7 +68,17 @@ module.exports = {
     // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-extraneous-dependencies.md
     // paths are treated both as absolute paths, and relative to process.cwd()
     'import/no-extraneous-dependencies': ['error', {
-      devDependencies: ['spec/**', 'test/**', 'tests/**', '**/__tests__/**'],
+      devDependencies: [
+        'spec/**', 
+        'test/**', 
+        'tests/**', 
+        '**/__tests__/**',
+        '**/webpack.config.js',
```

Comment:
I ever see some usage case like:

`examples/webpack.config.js` 

or 

```
webpack/webpack.config.js
webpack/webpack.config.dev.js
webpack/webpack.config.prod.js
webpack/webpack.config.test.js
```

But I can take those changes if it is too much.